src/navigations/HomeBottomTabs.js

```diff
@@ -14,13 +14,15 @@ import MemoNews from '../assets/icon/News';
 import MemoProfileIcon from '../assets/icon/Profile';
 import SignedChat from '../assets/icon/SignedChat';
 import StorageUtils from '../utils/storage';
+import dimen from '../utils/dimen';
 import useCoreChatSystemHook from '../hooks/core/useCoreChatSystemHook';
 import usePushNotificationHook from '../hooks/core/push-notification/usePushNotificationHook';
 import useRootChannelListHook from '../hooks/screen/useRootChannelListHook';
 import {COLORS} from '../utils/theme';
 import {ChannelListScreen, FeedScreen, NewsScreen, ProfileScreen} from '../screens';
 import {InitialStartupAtom, otherProfileAtom} from '../service/initialStartup';
 import {Loading} from '../components';
+import {fonts, normalizeFontSize} from '../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `dimen` and `normalizeFontSize` have been added. These are likely to be used for setting dimensions and font sizes in a way that scales well across different screen sizes. The order of imports has also been adjusted for better readability.
```diff
+ import dimen from '../utils/dimen';
+ import {fonts, normalizeFontSize} from '../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->